src/ProgressOnderwijsUtils/SchemaReflection/DatabaseDescription.cs

```diff
@@ -41,9 +41,10 @@ public sealed class DatabaseDescription
     readonly ILookup<DbObjectId, ForeignKey> fksByReferencedParentObjectId;
     readonly ILookup<DbObjectId, ForeignKey> fksByReferencingChildObjectId;
 
-    public DatabaseDescription(
+    DatabaseDescription(
```

Comment:
Or does this makes it a major release?

Reply:
We actually use this constructor in the CustomRapportenTest

Reply:
(i.e. technically yes, but you'll also need to solve the breaking change)


For stuff like this, just use `progress_with_utils_manually_checked_out.sln` and try to compile.